SuperBuild/External_python.cmake

```diff
@@ -113,6 +113,7 @@ if((NOT DEFINED PYTHON_INCLUDE_DIR
       -DBUILD_SHARED:BOOL=ON
       -DBUILD_STATIC:BOOL=OFF
       -DUSE_SYSTEM_LIBRARIES:BOOL=OFF
+      -DSRC_DIR:PATH=${python_SOURCE_DIR}
```

Comment:
You could also set `-DDOWNLOAD_SOURCES:BOOL=OFF` and `-DINSTALL_WINDOWS_TRADITIONAL:BOOL=OFF`

That way, the configuration/build of python would be more robust to change in default option. See https://github.com/python-cmake-buildsystem/python-cmake-buildsystem/commit/d40eb36424b264f5d97fe582479f181e00540dde